pkg/asset/manifests/cloudproviderconfig.go

```diff
@@ -34,7 +34,7 @@ var (
 )
 
 const (
-	cloudProviderConfigDataKey = "config"
+	cloudProviderConfigDataKey = "cloud.conf"
```

Comment:
if we merge https://github.com/openshift/machine-config-operator/pull/1973 I don't think we need this change.